frontend/packages/noobaa-storage-plugin/integration-tests/tests/backingStoreTest.scenario.ts

```diff
@@ -0,0 +1,64 @@
+import { execSync } from 'child_process';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { $, ExpectedConditions as until, browser } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  operatorsPage,
+  ocsOperator,
+  BackingStoreHandler,
+  Providers,
+  createBackingStoreLink,
+} from '../views/createBS.view';
+
+const NAME_BS = `${testName}-bs`;
+const NS = 'openshift-storage';
+
+const bsHandler = new BackingStoreHandler(NAME_BS);
+
+describe('Tests creation of Backing Store', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/ns/${NS}`);
+  });
+
+  beforeEach(async () => {
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await operatorsPage();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(ocsOperator));
+    await ocsOperator.click();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(createBackingStoreLink));
+    await createBackingStoreLink.click();
```

Comment:
Couple lines of code could be saved here by using `await click(element)` method from `@console/shared/src/test-utils/utils` instead of 
```
await browser.wait(until.visibilityOf(element));
await element.click()
```
But just a suggestion.